src/app/query-selector/layout.tsx

```diff
@@ -0,0 +1,18 @@
+'use client';
+
+import { AppShell, Text } from '@mantine/core';
+import classes from './page.module.css';
+
+export default function QuerySelectorLayout({ children }: { children: React.ReactNode }) {
+  return (
+    <AppShell padding="md">
+      <AppShell.Header className={classes.header}>
+        <Text component="a" href="/query-selector" className={classes.title}>
+          Bulk-export-app
+        </Text>
+      </AppShell.Header>
+      <AppShell.Main className={classes.mainPage}>{children}</AppShell.Main>
+      <AppShell.Footer></AppShell.Footer>
```

Comment:
```suggestion
      <AppShell.Footer />
```
I won't put a comment for every instance of this lol but an easy why to make code more concise is doing this if your component doesn't have any children!